docs/mergify_engine_gendoc.py

```diff
@@ -63,11 +63,13 @@ def write_redirect(app, pagename, templatename, context, doctree):
     url = f"https://docs.mergify.io/{pagename}/"
     os.makedirs(os.path.join(app.outdir, pagename), exist_ok=True)
     with open(os.path.join(app.outdir, pagename + ".html"), "w") as f:
-        f.write(f"""<head>
+        f.write(
+            f"""<head>
   <meta http-equiv="refresh" content="0; URL={url}">
   <link rel="canonical" href="{url}">
 </head>
-""")
+"""
+        )
```

Comment:
Unrelated change caused by the `tox -e black` command

Reply:
I will fix the tox target